Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -1457,6 +1458,12 @@ class IUserGroupsSettingsSchema(Interface):
     )
 
 
+def validate_twitter_username(value):
+    if value and value.startswith('@'):
+        raise Invalid(u'Twitter username should not include the "@" prefix character.')  # noqa: E501
```

Comment:
you can easily break this into two lines, so no need to add a noqa, exceptions to the rules are for that, for exceptions, not just because I don't want to split a sentence in two.